File: TransCoder_tokenized_test_set_functions/java/STRING_CONTAINING_FIRST_LETTER_EVERY_WORD_GIVEN_STRING_SPACES.java
static String firstLetterWord ( String str ) {
  String result = "";
  boolean v = true;
  for ( int i = 0;
  i < str . length ( );
  i ++ ) {
    if ( str . charAt ( i ) == ' ' ) {
      v = true;
    }
    else if ( str . charAt ( i ) != ' ' && v == true ) {
      result += ( str . charAt ( i ) );
      v = false;
    }
  }
  return result;
}

File: TransCoder_tokenized_test_set_functions/cpp/STRING_CONTAINING_FIRST_LETTER_EVERY_WORD_GIVEN_STRING_SPACES.cpp
string firstLetterWord ( string str ) {
  string result = "";
  bool v = true;
  for ( int i = 0;
  i < str . length ( );
  i ++ ) {
    if ( str [ i ] == ' ' ) v = true;
    else if ( str [ i ] != ' ' && v == true ) {
      result . push_back ( str [ i ] );
      v = false;
    }
  }
  return result;
}